### ISSUE TITLE
AirPods only show "Headset" and not "Hands Free"

### ISSUE THREAD

**Issue by aliasbody** [Rayzeq/quick-settings-audio-panel#33]
When I use this extension with any other BT Headet with mic it shows two entries:
- <BT_HEADSET_NAME> Headset
- <BT_HEADSET_NAME> Hands Free

But with the Airpods it only shows:
![image](https://github.com/Rayzeq/quick-settings-audio-panel/assets/9455122/05de1989-73b8-494f-8be1-a3a6f723f855)

Even known in the Gnome Audio Output list the Hands Free is there

![image](https://github.com/Rayzeq/quick-settings-audio-panel/assets/9455122/95ae3197-69a5-4787-803c-83672abd3000)

When I select it on the Gnome List it doesn't show as selected in the extension:
![image](https://github.com/Rayzeq/quick-settings-audio-panel/assets/9455122/29f70562-e680-4d88-944d-3b7b19497ced)

![image](https://github.com/Rayzeq/quick-settings-audio-panel/assets/9455122/c2abf39e-cc62-4ef2-9451-33a1dab375a4)

**Comment by aliasbody** [comment]
> Does this bug occurs without the extension ? Given that I'm not modifying this component (only moving it around), I'm almost certain that it's a gnome-shell issue, or an issue with another extension.

Which part of the issue? The lack of the Hands Free or the crash when clicking on the Headset when the Hands Free was selected via the interface? (I imagine this last one is not related to the extension).

Sorry for the dumb question.

**Comment by Rayzeq** [comment]
Both part of the issue, like I said I'm not modifying this component so I don't think I break anything (even though everything is possible with gnome shell extensions)

**Comment by aliasbody** [comment]
You're right, this issue seems to also happen without the extension. I'll report it in the Gnome Shell issue tracker. Sorry for the inconvenience.
